Petits-debuts-ui/src/Components/Trending/Trending.jsx

```diff
@@ -24,9 +27,7 @@ export default function Trending({
   const { trending, setTrending } = useContext(TrendingContext);
   const { productContext } = useContext(ProductContext);
   const { TotalOther, setTotalOther } = useContext(TotalOtherContext);
-  //start trending
-
-  //this function updates the local storage for trending
+  const thresholdMinimumScore = 70;
```

Comment:
Can we make `thresholdMinimumScore` capitalized?

Reply:
i don't understand
you mean THRESHOLDMIMINUMSCORE?

Reply:
Something like THRESHOLD_MINIMUM_SCORE. This is a common pattern for constants. It's called snake case

Reply:
okay